libs/@blockprotocol/graph/src/codegen/postprocess/generate-block-link-target-aliases.ts

```diff
@@ -0,0 +1,50 @@
+import { mustBeDefined } from "../../shared/util/must-be-defined.js";
+import {
+  typedEntries,
+  typedValues,
+} from "../../shared/util/typed-object-iter.js";
+import { PostprocessContext } from "../context/postprocess.js";
+import { entityDefinitionNameForEntityType } from "../shared.js";
+
+export const generateBlockLinkTargetAliases = (
+  context: PostprocessContext,
+): void => {
+  context.logDebug("Generating block entity types to generated files");
+
+  for (const [fileName, { blockEntity }] of typedEntries(
+    context.parameters.targets,
+  )) {
+    if (blockEntity) {
+      const type = mustBeDefined(context.allTypes[blockEntity]);
+      const typeName = entityDefinitionNameForEntityType(type.title);
+
+      /* @todo - make this more robust */
+      const entityLinkAndTargetsIdentifier = `${typeName}OutgoingLinksAndTargets`;
+
+      if (
+        typedValues(context.filesToDefinedIdentifiers).find(
+          (definedIdentifier) =>
+            definedIdentifier.has(entityLinkAndTargetsIdentifier),
+        )
+      ) {
+        context.logDebug(
+          `Generating block entity links and targets alias ${typeName} for file ${fileName}`,
+        );
```

Comment:
I threw this together somewhat quickly to get this PR up because it had been lagging. I don't like this, it could potentially be made less gross by at least defining this template somewhere shared but I'd really like to not rely on checking if this thing has been created already or manually recreating the identifier for it.

Perhaps it needs another map on the context 🙃